components/src/md3/Drawer/DrawerNavItem.tsx

```diff
@@ -180,6 +181,7 @@ export const DrawerNavItem: React.FC<DrawerNavItemProps> = (props) => {
     const previousActive = usePrevious(activeItem || '');
 
     // approximating primary[200] but we don't have access to it directly from the theme
+    /* eslint-disable-next-line @typescript-eslint/no-unused-vars */ //TODO: Do we need this variable?
```

Comment:
You can take this thing out, I think it MIGHT have been used to calculate drawer item background in the past, but in Material Design 3, we are using "primary container" for that